roxctl/common/tls.go

```diff
@@ -4,57 +4,15 @@ import (
 	"context"
 	"crypto/tls"
 	"crypto/x509"
-	"net"
 	"os"
 
 	"github.com/pkg/errors"
 	"github.com/stackrox/rox/pkg/clientconn"
 	"github.com/stackrox/rox/pkg/mtls"
 	"github.com/stackrox/rox/pkg/netutil"
-	"github.com/stackrox/rox/pkg/tlscheck"
 	"github.com/stackrox/rox/roxctl/common/flags"
-	"github.com/stackrox/rox/roxctl/common/logger"
-	"google.golang.org/grpc/codes"
-	"google.golang.org/grpc/status"
 )
 
-const errorMsg = `The remote endpoint failed TLS validation: %v
-
-  Do one of the following:
```

Comment:
Do we drop this message now? Is that intentional?

Reply:
I intended to remove the message, because I think if the error says about invalid certificate, it's clear what one need to do.